src/layout/graph_tree.rs

```diff
@@ -12,6 +12,14 @@ use rustwlc::WlcView;
 
 use layout::{Container, ContainerType};
 
+#[derive(Clone, Copy, Debug)]
+pub enum GraphError {
+    /// These nodes were not siblings.
+    NotSiblingsError(NodeIndex, NodeIndex),
```

Comment:
Nit: The other error is just named `NotParent` (which is easier because you're gonna need to write the `GraphError::` anyway)